### PR TITLE
Bluetooth: controller: Fix conn update to be cacheable

### PR DESCRIPTION
**Pull request by cvinayak**
Fix connection update procedure to be cacheable if any other
local or remote control procedure is in progress.

Signed-off-by: Vinayak Kariappa Chettimada <vich@nordicsemi.no>

### CODE REVIEW

**Comment by ppryga** [comment]
The issue was observed when perihperal device tried to start the update connection parameters procedure in time shorter than 1s after connection is established. To test that following PR has to be accepted: https://github.com/zephyrproject-rtos/zephyr/pull/19278. 

I've tested the fix and it works as expected.

**Review by ppryga-nordic — APPROVED** [review]

**Comment by cvinayak** [comment]
Added DNM, I want to test some more, give me time until 27/09/2019

**Comment by cvinayak** [comment]
@carlescufi this needs review, approval and backport to v1.14-branch

File: subsys/bluetooth/controller/ll_sw/ctrl.c [hunk 1]
```diff
@@ -8811,7 +8793,14 @@ static void event_connection_prepare(u32_t ticks_at_expire,
 	 * Encryption setup.
 	 */
 	if ((conn->llcp_ack == conn->llcp_req) && !conn->pause_rx) {
-		if (conn->llcp_feature.ack != conn->llcp_feature.req) {
+		if (conn->llcp_cu.ack != conn->llcp_cu.req) {
+			/* Previous event will be stop in the switch clause
```
[the remark this answers is not in the document]
Comment (cvinayak) [reply subsys/bluetooth/controller/ll_sw/ctrl.c]: at line 8871, before handling case LLCP_CONN_UPD.

**Review by joerchan — APPROVED** [review]

**Review by carlescufi — APPROVED** [review]

**Pull request merged**